docs/imagecustomizer/api/configuration/uki.md

```diff
@@ -69,3 +69,91 @@ os:
 ```
 
 Added in v0.8.
+
+## reinitialize [string]
+
+Controls the behavior when customizing an image that already contains UKI files.
+
+optional.
+
+This field is relevant when customizing an existing image that already has UKI
+files in the `/boot/efi/EFI/Linux` directory. It determines whether to preserve
+the existing UKI files or regenerate them during the customization process.
+
+Supported options:
+
+- (unspecified): When `reinitialize` is not specified, the tool generates new
+  UKI files based on the kernel and initramfs files present in the image. The
```

Comment:
Could we have a specific default value for reinitialize? It would be easier to understand what the default behavior is rather than calling this (unspecified). See pxe.initramfsType config documentation for a good example of this format (string enum value).